### DISCUSSION TITLE
Proposal: "Restore GRUB" Tool for Linux Mint

### DISCUSSION THREAD

**Discussion by silverhadch** [linuxmint/discussions#699] [Ideas]
Dual-booting with Windows often results in GRUB being overwritten during updates, and a misconfigured GRUB in /etc prevents generating a new grub.cfg, as update-grub will fail with errors.

Solution: Include a Restore GRUB tool in the Linux Mint Live CD or as a standalone app.

Key Features:

Automatically detect and reinstall GRUB.

Fix broken configs to allow grub.cfg generation.

Provide a simple interface for one-click restoration.

This tool would streamline recovery, especially for dual-boot users and thinkerers, making Linux Mint more beginner-friendly.

**Comment by claudiux** [comment]
This tool already exists: Run a live-DVD of Linux Mint and use "Boot Repair" in the Administration menu.

**Reply by silverhadch** [reply]
Never knew that! Thanks!!!
